com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -178,6 +183,26 @@ internal set
         /// </summary>
         public bool DontDestroyWithOwner;
 
+        public InterestSettings InterestSettingsOverride;
+        public InterestSettings InterestSettings
+        {
+            get
+            {
+                InterestSettings result = null;
+                if (InterestSettingsOverride)
+                {
+                    result = InterestSettingsOverride;
+                }
+                else if (NetworkManager.InterestSettings)
+                {
+                    result = NetworkManager.InterestSettings;
+                }
+
+                return result;
+            }
+            set => InterestSettingsOverride = value;
```

Comment:
This is a bit strange. We have a property modify another public field with its setter? Maybe InterestSettingsOverride should be private/internal?